app/views/layouts/application.html.haml

```diff
@@ -28,3 +27,8 @@
 
   %body{ class: add_rtl_body_class(body_classes) }
     = content_for?(:content) ? yield(:content) : yield
+    = javascript_include_tag '/piwik'
+    %noscript 
+      %p 
+        %img(src="https://analytics.remote-shell.net/piwik.php?idsite=2&rec=1" style="border:0;" alt="")
```

Comment:
second item of the TODO and in the commit logs ( urls are hardcoded right now ).

As I see some interest, working on dynamic for those parameters !